### PR TITLE
Remove golang/protobuf/jsonpbtest dep

### PR DESCRIPTION
**Pull request by johanbrandhorst**
With 1.4.0 the golang/protobuf json testing protofiles
are being moved to [an internal package][1]. While this is
technically a breaking change, I doubt the maintainers
are going to reintroduce it, so I've copied the necessary
parts to our own testing protofile.

This allows users of this package and, more importantly,
grpcui to try 1.4.0 without this package having to migrate yet.
When it comes to migrating to 1.4.0, this should also make the
transition easier.

[1]: https://github.com/golang/protobuf/tree/v1.4.0-rc.4/internal/testprotos

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module github.com/fullstorydev/grpcurl
 
+go 1.14
```
Comment (jhump) [inline go.mod:3]: Mind removing this? We _just_ finished a migration off of App Engine (go111 runtime), but we haven't yet hoisted ourselves off of Go 1.11. And 1.11 chokes on these.

Comment (jhump) [reply go.mod:3]: Hmmm, well CI is green, so I guess I am wrong. Maybe it was fixed in a later point release of Go 1.11...

**Comment by fenollp** [comment]
@fullstorydev @jhump Could you cut a release with this? Meanwhile we're using go mod edit to work around this mod update failure:
```
go: finding module for package github.com/golang/protobuf/jsonpb/jsonpb_test_proto
github.com/my/packge imports
	github.com/fullstorydev/grpcurl tested by
	github.com/fullstorydev/grpcurl.test imports
	github.com/golang/protobuf/jsonpb/jsonpb_test_proto: module github.com/golang/protobuf@latest found (v1.4.0), but does not contain package github.com/golang/protobuf/jsonpb/jsonpb_test_proto
```

**Comment by jhump** [comment]
@fenollp: https://github.com/fullstorydev/grpcurl/releases/tag/v1.5.1

**Pull request merged**